skyrim-platform/src/platform_se/skyrim_platform/EventsApi.cpp

```diff
@@ -165,10 +165,11 @@ class Hook
         return;
       }
 
-      return SkyrimPlatform::GetSingleton().AddUpdateTask([=] {
-        std::string s = eventName;
-        HandleEnter(owningThread, selfId, s);
-      });
+      return SkyrimPlatform::GetSingleton().AddUpdateTask(
+        [this, owningThread, selfId, eventName] {
```

Comment:
Self-question

Does `[=]` infer `*this` or `this`?

Reply:
https://en.cppreference.com/w/cpp/language/lambda

> The current object (*this) can be implicitly captured if either capture default is present. If implicitly captured, it is always captured by reference, even if the capture default is =. The implicit capture of *this when the capture default is = is deprecated. (since C++20)